backend/src/api/endpoints/user.py

```diff
@@ -75,6 +79,7 @@ async def update_by_id(
     response_model=Optional[UserRead],
     summary="Removes the User",
     description="Removes the User with the given ID if it exists",
+    dependencies=[is_authorized(ActionType.DELETE, crud_user, "user_id")],
 )
 async def delete_by_id(
     *, db: Session = Depends(get_db_session), user_id: int
```

Comment:
Is it on purpose that the delete route is not protected (yet)?

Reply:
I don't understand, line 82 has the protection dependency:
```py
dependencies=[is_authorized(ActionType.DELETE, crud_user, "user_id")],
```

Did you mean another routhe?